--- log4cpp-1.0/src/BasicLayout.cpp.orig	2006-09-30 08:03:20.000000000 +0200
+++ log4cpp-1.0/src/BasicLayout.cpp	2008-09-16 19:53:32.147768924 +0200
@@ -14,6 +14,7 @@
 #ifdef LOG4CPP_HAVE_SSTREAM
 #include <sstream>
 #endif
+#include <memory>
 
 namespace log4cpp {
 
--- log4cpp-1.0/src/PatternLayout.cpp.orig	2007-08-28 09:54:12.000000000 +0200
+++ log4cpp-1.0/src/PatternLayout.cpp	2008-09-16 19:55:42.417534061 +0200
@@ -24,6 +24,7 @@
 #include <iomanip>
 #include <ctime>
 #include <cmath>
+#include <cstdlib>
 #include "Localtime.hh"
 
 #ifdef LOG4CPP_HAVE_INT64_T
