--- log4cpp-0.3.4b/configure.in.orig	2002-10-29 22:40:37.000000000 +0100
+++ log4cpp-0.3.4b/configure.in	2003-12-21 21:40:15.760800432 +0100
@@ -69,8 +69,8 @@
 # ----------------------------------------------------------------------------
 
 # for RemoteSyslogAppender
-AC_CHECK_LIB(socket,socket, LIBS="-lsocket $LIBS",,)
-AC_CHECK_LIB(nsl,gethostbyname, LIBS="-lnsl $LIBS",,)
+AC_SEARCH_LIBS(socket,socket)
+AC_SEARCH_LIBS(gethostbyname,nsl)
 
 # idsa_test
 if test "x$with_idsa" = xyes; then
