--- log4cpp/m4/BB_ENABLE_REMOTE_SYSLOG.m4.orig	2023-03-12 17:46:57.000000000 +0100
+++ log4cpp/m4/BB_ENABLE_REMOTE_SYSLOG.m4	2025-04-13 21:49:00.014472635 +0200
@@ -4,8 +4,8 @@ AC_ARG_ENABLE(remote-syslog, [  --disabl
 
 # for RemoteSyslogAppender
 if test "x$enable_remote_syslog" = xyes; then
-    AC_CHECK_LIB(socket,socket, LIBS="-lsocket $LIBS",,)
-    AC_CHECK_LIB(nsl,gethostbyname, LIBS="-lnsl $LIBS",,)
+    AC_SEARCH_LIBS(socket,socket)
+    AC_SEARCH_LIBS(gethostbyname,nsl)
 else
     AC_DEFINE(DISABLE_REMOTE_SYSLOG,1,[define if RemoteSyslogAppender is disabled])
 fi
